calculator.js

```diff
@@ -1,28 +1,61 @@
 
-const exampleAdditionInput = {
-  num1: 3,
-  num2: 5,
-  operation: 'add',
-}
-// '3 + 5 = 8'
+const calculateAdd = function(input) {
```

Comment:
Great work encapsulating the operation functionality into functions. If all you are doing is console.log, you do no necessarily need a return. You might consider performing the `console.log` and returning the result.